app/utils/timeDifference.ts

```
@@ -0,0 +1,29 @@
+
+    const start = new Date(parseStartDate.data);
+    const end = new Date(parseEndDate.data);
+
+    const differenceInMilliseconds = Math.abs(end.getTime() - start.getTime());
+
+    const differenceInMinutes =
+      differenceInMilliseconds / MILLISECONDS_IN_A_MINUTE;
+
+    return differenceInMinutes;
+  } catch (e) {
+    throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)